### PR TITLE
Volvo day 12 13 & 14 beta test and misc edits

### PR DESCRIPTION
**Pull request by Volvo-hackangel**
work in progress :necoarc:

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 1]
```diff
@@ -295,7 +295,7 @@ Akiha jumps straight to the heart of the matter.
 [sha:0dbede9a5e1c181ae7b78fda16799f64ca76b38b]{
 -- Page 19, Offset 31725.
 -- 驚きと緊張で喉が動かない。
-My throat seizes up from a combination of shock and nerves.
+My throat seizes up from a combination of shock and nervousness.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1.txt:298]: revert

File: script/Arcueid/Day 4/04_00_ARC04_6_2D.txt [hunk 2]
```diff
@@ -969,7 +969,7 @@ Sensing danger, I back away.
 [sha:8e422e9f871066decf89fecaf907f6e08ec3a824]{
 -- Page 634, Offset 7266.
 -- しかし、この大猫に俺の脅しが効くはずもなく、アルクェイドは興味津々とばかりに近寄ってくる。
-However, my threat was entirely ineffective against the big cat in front of me, and Arcueid continues to prowl nearer with a keen curiosity.
+However, my threat was entirely ineffective against the big cat in front of me, and Arcueid continues to prowl closer with a keen curiosity.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2D.txt:972]: revert

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt [hunk 3]
```diff
@@ -471,7 +471,7 @@ As I understand it, vampires that have been alive for a long time aren't very fu
 [sha:28abac3df6ca575115ad2774eb62e39807395543]{
 -- Page 546, Offset 17937.
 -- その手足を動かすごとに自分の力を消費していては話にならない。
- So it would go against that purpose if they needed to expend their own energy on those underlings.
+So it would go against that purpose if they needed to expend their own energy on those underlings.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:474]: undo this. it will make the line render weird

i took it out of my pr

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt (hunk 3 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:474]: huh ok. what a weird error

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt [hunk 4]
```diff
@@ -639,7 +639,7 @@ Even though we're helplessly slaughtered as food...?
 [sha:fc1cd847a33664800a8f8c005f14eb16834edd38]{
 -- Page 548, Offset 25683.
 -- 人間は様々な魔術、神秘学、式典儀礼を基に組織体系を作り、人間以外の霊長類を排除しはじめた。
-Humans created organizations based on various kinds of magecraft, occultism, and ritualism, and then began eliminating non-human Paragons.
+Humans created organizations based on various kinds of magecraft, occultism, and ritualism, and then began eliminating nonhuman Paragons.
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_5C.txt:642]: original is correct, non-human is being used as an adjective not noun.

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt [hunk 5]
```diff
@@ -736,7 +736,7 @@ If the Executors deployed here had been theirs,
 // If vampires are primates does that imply the existence of vampire monke
 // no more vampire monke
 "The reason that Executor, Noel, hasn't done anything until now is probably because she was waiting for a chance to deal with me and the Dead Apostle at the same time.
- After all, to the Burial Agency, any non-human Paragon is 'evil' in and of itself."
+ After all, to the Burial Agency, any nonhuman Paragon is 'evil' in and of itself."
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_5C.txt:739]: same here

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 6]
```diff
@@ -1026,7 +1026,7 @@ They tend to be discriminated against―――branded as 'psychics' or 'changeli
 -- Page 732, Offset 26262.
 -- 特異能力ってのは肉体的なものだからな、家系―――血筋で遺伝する。ロアは自分の新しい肉体にそういった『人間ではないもの』の家系を選ぶって話だ」
 Special abilities like these are physiological, so they're passed down through the family―――through their bloodline.
- Roa tends to pick members of these kinda 'non-human' families for his new body."
+ Roa tends to pick members of these kinda 'nonhuman' families for his new body."
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:1029]: same here

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 7]
```diff
@@ -545,7 +545,7 @@ Turning to look me straight in the eye, Akiha begins to speak.
 -- Page 36, Offset 31773.
 -- 「兄さん。遠野の家が特別な血族だという事はあの人たちに聞いたでしょう？　信じられないでしょうけど、遠野の血は人間以外の血が混ざっているんです。
 "Nii-san. You've heard from those people that the Tohno family is a special bloodline, correct?
- This may be hard to believe, but the Tohno blood is mixed with that of something non-human.
+ This may be hard to believe, but the Tohno blood is mixed with that of something nonhuman.
```
Comment (yk1512) [inline script/Arcueid/Day 13/13_00_ARC13_1.txt:548]: same here

File: script/Arcueid/Day 5/05_00_ARC05_2.txt [hunk 8]
```diff
@@ -250,7 +250,7 @@ Their deaths are the fault of the entire human race, which has somehow evolved t
 [sha:606fd77a18969d106fc5b1d1519ad00a97c2bf84]{
 -- Page 190, Offset 9055.
 -- 人が人を殺したというのなら、確かに人間社会での罪になる。けれどこれは人間と人間外のものに起きた自然淘汰の一環よ。あなた個人の責任なんて思うのは<傲慢|ごうまん>だわ」
-If a human kills another human, that would certainly be considered a crime in human society. But this is a part of natural selection between humans and non-humans. It would be arrogant to hold yourself personally responsible."
+If a human kills another human, that would certainly be considered a crime in human society. But this is a part of natural selection between humans and nonhumans. It would be arrogant to hold yourself personally responsible."
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_2.txt:253]: this one is fine, don't change it.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt [hunk 9]
```diff
@@ -229,7 +229,7 @@ Would you be willing to take off your glasses while you're accompanying me?"
 [sha:2c84f38f88d18404507329a76a1ddf1cbe880415]{
 -- Page 444, Offset 17676.
 -- わたしは人間と人間じゃないものの気配を感じ取れる。
-I can sense the presence of human and non-human beings.
+I can sense the presence of human and nonhuman beings.
```
Comment (yk1512) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:232]: change

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt [hunk 10]
```diff
@@ -399,7 +399,7 @@ These Twenty-Seven Ancestors represent the apex of the Dead Apostles, right?"
 // could use some work still, but this is preferrable.
 "Correct.
  They're ancient Dead Apostles who once broke free from their role as emergency food,
- winning their independence and establishing themselves as completely different beings.
+ winning their independence, and establishing themselves as completely different beings.
```
Comment (yk1512) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:402]: remove

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 11]
```diff
@@ -56,7 +56,7 @@ She's neither a fool nor a bad person.
 [sha:e70dd5c28b737ae56ad8f139546f68529b1a8763]{
 -- Page 580, Offset 12365.
 -- 俺たちとは違う価値観で動いている、人間ではない生き物なんだ。
-She's a non-human creature who operates on a completely different set of values than us,
+She's a nonhuman creature who operates on a completely different set of values than us,
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:59]: change

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 12]
```diff
@@ -633,7 +633,7 @@ I am not talking about magecraft, those mysteries that can be learned, but about
 [sha:b056e761f1b12980a6c138839bd64b45c9246647]{
 -- Page 200, Offset 22634.
 -- 「特異能力というのは遺伝するケースが多いですから、ロアは自分の新しい肉体にそういった『人間ではないもの』の家系を選ぶこともあったと言います」
-"As supernatural powers are often hereditary, there has been cases where Roa chose his new body among lineages of 'non-humans'."
+"As supernatural powers are often hereditary, there has been cases where Roa chose his new body among lineages of 'nonhumans'."
```
Comment (yk1512) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:636]: this is fine

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 13]
```diff
@@ -409,7 +409,7 @@ That Dead Apostle put the city under his complete control, killed all the Execut
 [sha:589669f57faa5e8206ea87586df517431a67bcb1]{
 -- Page 408, Offset 23185.
 -- わかる？　吸血鬼を殺す事だけに特化した非人間。
-Do you understand? She's a non-human whose sole specialization is in killing vampires.
+Do you understand? She's a nonhuman whose sole specialization is in killing vampires.
```
Comment (yk1512) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:412]: this is fine

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt (hunk 4 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 10/10_00_ARC10_5C.txt:642]: you sure i just went with what ngrams said was common :necoarc:
image

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt [hunk 14]
```diff
@@ -175,7 +175,7 @@ Instead, she turns away from the vampire and takes my fallen body in her arms.
 [sha:e42d858893a7770c79da0561bc38d30539c7febb]{
 -- Page 89, Offset 30787.
 -- 「ほう。私との因果を断つより、そのニセモノのほうが大事という事か。
-"%{i}Hoho%{/i}. Are you really prioritizing that impostor over severing the karmic ties that bind the two of us together?
+"%{i}Hoh%{/i}. Are you really prioritizing that impostor over severing the karmic ties that bind the two of us together?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 12/12_00_ARC12_2B.txt:178]: Maybe just drop the italics and make this 
> Oh? Are you really...

File: script/Arcueid/Day 13/13_00_ARC13_4.txt [hunk 15]
```diff
@@ -1375,7 +1375,7 @@ By shaking her head.
 What are you still afraid of?!
  Listen to me.
  Do you remember what you said a couple days ago?
- You asked me if I'd still eat birds or fish even if they had the same level of intelligence as me.
+ You asked me if I'd still eat birds or fish even if they had the same level of intelligence as mine.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_4.txt:1378]: revert

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt [hunk 16]
```diff
@@ -907,7 +907,7 @@ I suppose that's how delicate the lives of the diabolical are.
 [sha:3b5be2b31c0b8c7fe65abdf42550a10134fe6200]{
 -- Page 572, Offset 18010.
 -- たとえ不老不死だとしても、存在意義がなければ空気と変わらないんだもの」
-Even assuming that you were immortal, if your life has no meaning, then you're no different than the air."
+Even assuming that you were immortal, if your life has no meaning, then you're no different than air."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:910]: _maybe_ revert, not sure

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt [hunk 17]
```diff
@@ -332,7 +332,7 @@ That said, I am fundamentally different to them,
 -- そうね。基本的な蓄えだけで言えば、きっとヴローヴより優れている」
 // そうね said fairly succinctly, but can't think of a great equivalent phrase with that intonation.
 // the accumulation thing most probably talks about their "accumulation of time, blood, etc", things that denote how strong vampires get.
-How to put it.
+How to put it?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:335]: Not the inflection for a ? IIRC

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt [hunk 18]
```diff
@@ -528,13 +528,13 @@ What makes an Ancestor an Ancestor is the character of their blood."
...
-The truths they have each been crowned;
- The worldviews that underpin their thirst;
+the truths they have each been crowned;
+ the worldviews that underpin their thirst;
 }
 [sha:ded30ad443ac5e43d2d483b697e500c9c52eb5ad]{
 -- Page 609, Offset 18106.
 -- その血を巡らせるだけで惑星の物理法則を塗り替える特異点。
-The singularities that paint over the planet's physical laws just by circulating their blood.
+the singularities that paint over the planet's physical laws just by circulating their blood.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:537]: I prefer to keep all new lines starting with caps tbh

**Review by rschlaikjer — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt (hunk 4 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_5C.txt:642]: both are valid according to [wikipedia](https://en.wikipedia.org/wiki/Non-human), and nonhuman does seem to definitely be more common.

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt (hunk 10 above)
Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:402]: nah, i think this should stay

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt (hunk 18 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:537]: image
looks like this in game
should i bring down the "The worldviews" one so all of them start in a new line?

**Pull request merged**